src/main/java/uk/co/ramp/covid/simulation/place/CommunalPlace.java

```diff
@@ -87,14 +91,14 @@ public boolean checkVisit(Person cPers, int time, int day, boolean clockdown) {
                             if (!nPers.getInfectionStatus()) {
                                 boolean infected = nPers.infChallenge(this.transProb * this.sDistance);
                                 if (infected) {
-                                    stats.infectedPlace(this, nPers);
+                                    registerInfection(stats, cPers);
```

Comment:
Why has this has changed from nPers to cPers?

Reply:
Good spots, looks like bugs - I'll fix this up